feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/presentation/settings/PushSettingsViewModel.kt

```diff
@@ -245,4 +250,20 @@ class PushSettingsViewModel(
                 )
             }
     }
+
+    private fun disableNotificationsIfPushSettingsEmpty() {
+        pushSettingsState
+            .onEach { pushSettings ->
+                if (pushSettings?.settingsIsEmpty() == true) {
+                    pushEnabledState.value = false
+                }
+            }
+            .launchIn(this)
+    }
+
+    private suspend fun makeDefaultPushSettingsIfNeeded() {
```

Comment:
I suggest `setDefaultPushSettingsIfEmpty()` - `ifNeeded` is rather generic and requires diving into the method's implementation to understand what "needed" means) I would use it only if we cant come up with more specific name, e.g. it is hard to reflect the behavior in the name

Reply:
Agree, fixed!